### PR TITLE
Recommend Camelcasing queryparams

### PR DESCRIPTION
**Pull request by masoodusmani**
Also json schema file names

### CODE REVIEW

File: api-style-guide.md [hunk 1]
```diff
@@ -883,18 +884,18 @@ For more specific info on the query parameter usage, see [URI Standards](#uri-st
...
 - Fields that represent arrays SHOULD be named using plural nouns (e.g. authenticators-contains one or more authenticators, products-contains one or more products).
 
 <h2 id="enum-names">Enum Names</h2>
 
-Entries (values) of an enum SHOULD be composed of only upper-case alphanumeric characters and the underscore character, ( \_ ).
+Entries (values) of an enum SHOULD be composed of only upper and lower case alphanumeric characters.
 
-- `FIELD_10`
-- `NOT_EQUAL`
+- `inProgress`
```
Comment (aimed) [inline api-style-guide.md:897]: I think here we can stick to `FIELD_10`, that's a common thing for enums right?

File: patterns.md [hunk 2]
```diff
@@ -530,7 +530,7 @@ It is not necessary that an API support the updating of all attributes via a `PA
 
 See [Sample Input Validation Error Response](index.md#sampleresponse-multi) for examples of error handling.
 
-<h4 id="patch_examples">PATCH Examples</h4>
+<h4 id="patchExamples">PATCH Examples</h4>
```
Comment (aimed) [inline patterns.md]: mhh

File: api-style-guide.md (hunk 1 above)
Comment (masoodusmani) [reply api-style-guide.md:897]: yeah, but the enums in the api v2 yaml file seem to use a mix of camel case and all uppercase, does that mean that they should be updated as well?

File: patterns.md (hunk 2 above)
Comment (masoodusmani) [reply patterns.md]: Changed!

**Review by aimed — APPROVED** [review]
🥇

**Pull request merged**